### PR TITLE
Use `url` helper for security links

### PR DESCRIPTION
**Pull request by boesing**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### Description

Adding url Helper as suggested in https://github.com/laminas/getlaminas.org/pull/43#discussion_r416855212

### CODE REVIEW

File: templates/security/security.phtml [hunk 1]
```diff
@@ -57,8 +57,8 @@
                     A security advisory will be released on the Laminas Project
                     site detailing the vulnerability, as well as
                     recommendations for end-users to protect themselves.
-                    Security advisories will be listed at <a href="/security/advisories">https://getlaminas.org/security/advisories</a>,
-                    as well as via a <a href="/security/feed">feed</a> (which
+                    Security advisories will be listed at <a href="<?= $this->url('security.pages', ['action' => 'advisories']) ?>">https://getlaminas.org/security/advisories</a>,
```
Comment (michalbundyra) [inline templates/security/security.phtml:60]: I think we should use the helper also for the content - the text we have between `<a...>` and `</a>`.

Comment (froschdesign) [inline templates/security/security.phtml:60]: To output the full URL the option `force_canonical` can be used:

https://docs.laminas.dev/laminas-view/helpers/url/#fully-qualified-domain-name

Comment (weierophinney) [reply templates/security/security.phtml:60]: If you do, use the `serverurl()` helper to give it a fully qualified URL.

Comment (weierophinney) [reply templates/security/security.phtml:60]: OMG, I had no idea that feature existed! I've been doing all sorts of custom helpers to accomplish this!

Comment (boesing) [reply templates/security/security.phtml:60]: Oh, I knew that feature exists. But as there is a `mezzio` "url" helper aswell (which does not support that option) I did not add it. Thats kinda confusing tho.

Comment (boesing) [reply templates/security/security.phtml:60]: And thats actually which is being used, as we are not using laminas-view but the platesrenderer.

https://github.com/mezzio/mezzio-helpers/blob/5.3.0/src/UrlHelper.php#L112

The UrlHelper only provides the path.

But I can combine with the `ServerUrlHelper` instead to call 2 functions tho.

**Comment by michalbundyra** [comment]
Thanks, @boesing!

**Pull request merged**